core/api/src/config/schema.types.d.ts

```diff
@@ -36,7 +36,7 @@ type YamlSchema = {
     android: BuildNumberInput
     ios: BuildNumberInput
   }
-  rewards: {
+  quizzes: {
```

Comment:
this will break custom config in deployments (just to keep in mind before deploy)

Reply:
yup I know